pkg/options/jsonnet/testdata/pod-with-imports.builder.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: bundle.gke.io/v1alpha1
```

Comment:
How about you move all your testdata to /examples/jsonnet?

Reply:
Done